README.md

```diff
@@ -32,29 +32,29 @@ rails_best_practices works well only in ruby 1.9.3, 2.0.0, and 2.1.0 so far.  It
 
 ## Usage
 
-At the root directory of rails app
+At the root directory of a Rails app, run:
 
     rails_best_practices .
 
-or html output
+Or for HTML output:
 
     rails_best_practices -f html .
 
-By default rails_best_practices will do parse codes in vendor, spec, test and features directories.
+By default rails_best_practices will parse code in the `vendor`, `spec`, `test` and `features` directories.
 
-### Excluding folders
+### Excluding directories
```

Comment:
I changed this to 'directory' to be consistent with the previous usage. Also, directory is more correct - not all operating systems use the name 'folder'.